components/compositing/Cargo.toml

```diff
@@ -41,6 +41,8 @@ servo_geometry = { path = "../geometry" }
 servo_url = { path = "../url" }
 style_traits = { workspace = true }
 time = { workspace = true }
+tracing = { workspace = true }
+tracing-subscriber = { workspace = true }
```

Comment:
nit: do we need these tracing-subscriber dependencies outside of servoshell?

Reply:
we don't, only `tracing` is needed. Thanks.